drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -103,9 +170,19 @@ end
 --- and supports the OnOff cluster. This is done to bypass the
 --- BRIDGED_NODE_DEVICE_TYPE on bridged devices
 local function find_default_endpoint(device, component)
-  local eps = device:get_endpoints(clusters.OnOff.ID)
-  table.sort(eps)
-  for _, v in ipairs(eps) do
+  local switch_eps = device:get_endpoints(clusters.OnOff.ID)
+  local button_eps = device:get_endpoints(clusters.Switch.ID, {feature_bitmap=clusters.Switch.types.Feature.MOMENTARY_SWITCH})
+  local all_eps = {}
+
+  for _,v in ipairs(switch_eps) do
```

Comment:
I think we should have a clearer variable name than v, for everything below as well

Reply:
Updated to `ep` to make it clear that we are iterating through the endpoints